dhis-2/dhis-services/dhis-service-administration/src/test/java/org/hisp/dhis/dataintegrity/DataIntegrityYamlReaderTest.java

```diff
@@ -56,7 +56,7 @@ void testReadDataIntegrityYaml() {
 
     List<DataIntegrityCheck> checks = new ArrayList<>();
     readYaml(checks, "data-integrity-checks.yaml", "data-integrity-checks", CLASS_PATH);
-    assertEquals(72, checks.size());
+    assertEquals(73, checks.size());
```

Comment:
Be aware that if we merge both your PRs this will break. We should make this a more relaxed condition

Reply:
Roger that. Will fix in the merge but would be good to able to handle this independently.